src/lib/display.ts

```diff
@@ -0,0 +1,67 @@
+import { Display, Displayable } from '@salesforce/sfdx-scanner/lib/lib/Display';
+import {Spinner, Ux} from "@salesforce/sf-plugins-core";
+import {AnyJson} from "@salesforce/ts-types";
+
+
+export class VSCodeDisplay implements Display {
+    private readonly displayable: Displayable;
+	private readonly spinner: Spinner;
+	private readonly isVerboseSet: boolean;
```

Comment:
@stephen-carter-at-sf - I have been going back and forth on this. There is still some implementation now and have added tests to please the code-coverage. But don't see any value in actually using the display. Though that would have been the simpler approach as you had suggested, there is value in consuming the results from the runAction to make us move closer to how it would be once we have the core package in place. Anyways, I have removed all the unused and added some new.